### ISSUE TITLE
@ExpectedDatabase value attribute input two or more requests

### ISSUE THREAD

**Issue by bluepoet** [springtestdbunit/spring-test-dbunit#64]
I think the @ExpectedDatabase annotations added every table gon inconvenience . 
@DatabaseSetup Annotations as two or more requests to enter the value attribute.

**Comment by philwebb** [comment]
I'm afraid I'm not following what you mean, could you rephrase?

**Comment by bluepoet** [comment]
@philwebb I think the trouble is that "@ExpectedDatabase" annotations are used for every table. It would be nice if you could put more than one value in the value attribute with the "@DatabaseSetup" annotation.

**Comment by philwebb** [comment]
So you would like the `String value()` attribute of `@ExpectedDatabase` to be `String[] value()`?

Have you seen the `@ExpectedDatabases` annotation that was added to the latest release?

**Comment by bluepoet** [comment]
@philwebb  I have resolved to write @ExpectedDatabases like the following code to set the override attribute to a false ExpectedDatabase.

@ExpectedDatabases ({
    @ExpectedDatabase (AssertionMode =DatabaseAssertionMode.NON_STRICT, value ="a.xml"),                    @ExpectedDatabase (AssertionMode = DatabaseAssertionMode.NON_STRICT, value ="b.xml", override = false)})

Thank you @philwebb #86

**Comment by calle2010** [comment]
Hello, I think I ran into the same issue that @bluepoet encountered and worked around using override=false:

I found that @ExpectedDatabase is @Repeatable, but in a test that should fail it depended on the order of the annotations on the method:

```
@DatabaseSetup("setup_ParentSpring.xml")
@DatabaseSetup("setup_ChildSpring.xml")
@ExpectedDatabase(value = "setup_ChildSpring.xml", assertionMode = DatabaseAssertionMode.NON_STRICT_UNORDERED)
@ExpectedDatabase(value = "expect_ParentSpring_updated.xml", assertionMode = DatabaseAssertionMode.NON_STRICT_UNORDERED)
```

The test method is actually empty, so the expectation is that the annotation with expect_ParentSpring_updated.xml fails the test, since no update happens.

But if I change the order of the @ExpectedDatabase annotations, the test succeeds:

```
@DatabaseSetup("setup_ParentSpring.xml")
@DatabaseSetup("setup_ChildSpring.xml")
@ExpectedDatabase(value = "expect_ParentSpring_updated.xml", assertionMode = DatabaseAssertionMode.NON_STRICT_UNORDERED)
@ExpectedDatabase(value = "setup_ChildSpring.xml", assertionMode = DatabaseAssertionMode.NON_STRICT_UNORDERED)
```

By checking the code I found that verification stops if one annotation has the override flag set to true, which is the default for @ExpectedDatabase, see https://github.com/springtestdbunit/spring-test-dbunit/blob/8b38e19c5b720bf4fbdc50d2f7a9461381ea3d2c/spring-test-dbunit/src/main/java/com/github/springtestdbunit/DbUnitRunner.java#L162

In this case, the last annotation is evaluated first and, since override==true, stops further verification.

I think the concept needs refinement or this behavior should be documented.

Workaround is, like @bluepoet wrote, to set override=false on the annotations after the the first (or all, if no class level annotation has been done).

@ExpectedDatabases annotation doesn't help since it shows the same issue.
